addons/l10n_ad/models/account_tax.py

```diff
@@ -0,0 +1,22 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    l10n_ad_exempt_reason = fields.Selection(
+        selection=[
+            ("E1", "No exempt"),
```

Comment:
As it is a l10n module, you will need to add the translations in Spanish (I suppose?) in a `.po`.

Reply:
Ok, fixed in 0d279c1. Translations are not there yet, though.